osCam/user/models.py

```diff
@@ -5,5 +5,4 @@
 # Create your models here.
 # Using Django Built in User for Now, until we need profile
 
-# class UserProfile(User):
-
+# class UserProfile(User):
```

Comment:
should we just remove this file? or do we plan on using it eventually?

Reply:
I will probably never touch this, I think we should delete the app entirely. Everything that was going to live in here is in userconfig which i kinda thing should be renamed settings. 

but renaming something in django can be a whole task in itself lol

Reply:
I'm fine with either renaming to settings or leaving as is. I agree that settings seems more natural, but if it's a hassle then it might not be worth the effort.